db.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose')
+
+mongoose.connect('mongodb://localhost/a11y-scans', {
```

Comment:
I think we'll have to make this an environment variable so it can be set on Heroku. I'm happy to leave that until we are dealing with that situation, just to keep in mind.